cmd/cmd.go

```diff
@@ -161,11 +163,14 @@ func (o *options) Validate() error {
 		return errors.New("One of pod-query, --selector, --field-selector, --prompt or --stdin is required")
 	}
 	if o.selector != "" && o.resource != "" {
-		return errors.New("--selector and the <resource>/<name> query can not be set at the same time")
+		return errors.New("--selector and the <resource>/<name> query cannot be set at the same time")
 	}
 	if o.noFollow && o.tail == 0 {
 		return errors.New("--no-follow cannot be used with --tail=0")
 	}
+	if o.condition != "" && o.tail != 0 {
+		return errors.New("--condition is currently only supported with --tail=0")
+	}
```

Comment:
How about excluding `--no-follow` from the validation, since it doesn't accept `--tail=0` but can work with the `--tail` option?

```suggestion
	if o.condition != "" && o.tail != 0 && !o.noFollow {
		return errors.New("--condition is currently only supported with --tail=0 or --no-follow")
	}
```

Reply:
You're right. I made some tests and --no-follow indeed works ok with --condition.